src/python/grpcio/grpc/_cython/_cygrpc/completion_queue.pyx

```diff
@@ -74,7 +68,41 @@ cdef class CompletionQueue:
     with self.poll_condition:
       self.is_polling = False
       self.poll_condition.notify()
+    return self.interpret_event(event)
+
+  def pluck(self, records.OperationTag tag, records.Timespec deadline=None):
+    # Plucking a 'None' tag is equivalent to passing control to GRPC core until
+    # the deadline.
+    # TODO(atash) there's currently no way of getting a tag to pass to this
```

Comment:
Because there's a pull between having the function match the underlying core API as much as possible (that's what the Cython layer is supposed to do) and what it's being added for (which is a strict subset of the functionality it was meant for).